src/api/subscription/use-notifications-list.tsx

```diff
@@ -1,9 +1,55 @@
+import { useEffect, useState } from 'react';
 import { useAuthorizedSubscription } from '../../base';
 
+type TNotification = {
+    category: string;
+    id: number;
+    links: {
+        href: string;
+        rel: string;
+    }[];
+    message_key: string;
+    payload: string;
+    read: boolean;
+    removed: boolean;
+};
```

Comment:
We can reuse the existing `TSocketResponse` type to define our notifications:

```suggestion
import { TSocketResponse } from '../../types/api.types';

type TNotifications = TSocketResponse<'notifications_list'>['notifications_list']['messages'];
```
We can then use `TNotifications[number]` to represent a single notification object.

```typescript
notifications.sort((a: TNotifications[number], b: TNotifications[number]) => b.id - a.id);
```